content/docs/connect/choose-connection.md

```diff
@@ -0,0 +1,133 @@
+</Admonition>
+
+### Next, choose your connection type: direct or pooled
+
+You then need to decide whether to use direct connections or pooled connections (using PgBouncer):
+
+- **In general, use pooled connections whenever you can**
+
+  Pooled connections can efficiently manage high numbers of concurrent client connections, up to 10,000. This 10K ceiling works best for serverless applications and application-side connection pools that have many open connections, but infrequent and/or short transactions.
+
+- **Use direct (unpooled) connections if you need persistent connections**
+
+  If your application is focused mainly on tasks like migrations or administrative operations that require stable and long-lived connections, use an unpooled connection.
+
+<Admonition type="note">
+Connection pooling is not a magic bullet. PgBouncer can keep many client connections open (up to 10,000) concurrently, but only a limited number of these can be actively querying the PostgreSQL server at any given time: 64 active backend transactions per user-database pair, as set by the PgBouncer's `default_pool_size` setting. For example, the Postgres user `alex` can hold up to 64 backend connections to a single database at one time.
+</Admonition>
+
+For more information on these choices, see:
+
+- [Neon Serverless Driver](/docs/serverless/serverless-driver)
+- [Connection pooling](/docs/connect/connection-pooling)
+
+## Common Pitfalls
+
+Here are some key points to help you navigate potential issues.
+| Issue | Description |
+|----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Double pooling | Avoid using client-side pooling if you're using a pooled Neon connection (supported by PgBouncer). Just let Neon handle the pooling to prevent retaining unused connections on the client side. If you must use client-side pooling, make sure connections are released back to the client-side pool early enough to avoid conflicts with PgBouncer. |
+| Understanding limits | Don't confuse `max_connections` with `default_pool_size`.<br /><br />`max_connections` is the maximum number of concurrent connections allowed by Postgres and is determined by your [Neon compute size](/docs/connect/connection-pooling#connection-limits-without-connection-pooling).<br /><br />`default_pool_size` is the maximum number of backend connections or transactions that PgBouncer supports per user/database pair, which is set to 64 by default.<br /><br />Simply increasing your compute to get more `max_connections` may not improve performance if the bottleneck is actually on your `default_pool_size`. To increase your `default_pool_size`, contact [Support](/docs/introduction/support). |
```

Comment:
not sure about "backend connections".

Reply:
I think it helps clarify: 
- client connections = open connections between the application and pgbouncer
- backend connections = active transactions between pgbouncer and the postgres 

I think some of the customer confusion might come from where we describe both sides of the pgbouncer handoff as "concurrent connections". Using "client" and "backend" where we can might help. The terms seemsstandard in various postgres docs/posts, and we should probably update our pooling docs to define and use those those terms, and use them in new diagrams as well.

So basically, double down. :)
WDYT?

Reply:
ok! This should help -> "define and use those those terms, and use them in new diagrams as well"

Reply:
For now, added some small clarifications throughout.